bibletime.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 1c6235b3..d48434a0 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -314,6 +314,10 @@ CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/cmake/platforms/linux/bibletime.desk
+                "${CMAKE_CURRENT_BINARY_DIR}/bibletime.desktop" @ONLY)
+ INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/bibletime.desktop"
+         DESTINATION "${BT_SHARE_PATH}/applications/")
++CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/cmake/platforms/linux/info.bibletime.BibleTime.appdata.xml"
++               "${CMAKE_CURRENT_BINARY_DIR}/info.bibletime.BibleTime.appdata.xml" @ONLY)
++INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/info.bibletime.BibleTime.appdata.xml"
++        DESTINATION "${BT_SHARE_PATH}/appdata/")
+ 
+ 
+ IF(MSVC)
+diff --git a/cmake/platforms/linux/info.bibletime.BibleTime.appdata.xml b/cmake/platforms/linux/info.bibletime.BibleTime.appdata.xml
```

Comment:
This would be better as separate  file and `post-install`, unless it's backport from upstream.